OrderNotificatorClient/src/app/shared/time-selector/time-selector.component.ts

```diff
@@ -10,4 +12,43 @@ import { MatButtonToggleModule } from '@angular/material/button-toggle';
 })
 export class TimeSelectorComponent {
 
+  private subscription: Subscription = new Subscription();
+
+  constructor(private orderService: OrderService){}
+
+  ngOnInit(){
+
+  }
+
+  ngOnDestroy() {
+    if (this.subscription) {
+      this.subscription.unsubscribe();
+    }   
+  }
+
+  setPizzaDeliveryTime(time: number) : Date{
+    let time_ = parseInt(time.toString());
+    let currentDate = new Date();
+    currentDate.setMinutes(currentDate.getMinutes() + time_);
+    return currentDate;
```

Comment:
Name of method should be something like "calculatePizzaDeliveryTime" because we don't 'set' that  time anywhere. Just calculate and return.
Name of parameter should be 'minutesToAdd'. We should avoid naming like 'time_' with underscore.

Reply:
I agree with you.